src/dev/i18n/extract_code_messages.js

```diff
@@ -38,9 +38,17 @@ import { extractIntlMessages, extractFormattedMessages } from './extract_react_m
 export function isIntlFormatMessageFunction(node) {
   return (
     isCallExpression(node) &&
-    isMemberExpression(node.callee) &&
-    isIdentifier(node.callee.object, { name: 'intl' }) &&
-    isIdentifier(node.callee.property, { name: 'formatMessage' })
+    (
+      isIdentifier(node.callee, { name: 'formatMessage' }) ||
```

Comment:
__nit:__ please extend method JSDoc to include examples for new supported cases as well (e.g. `props.intl.formatMessage` and
```js
const { formatMessage }  = this.props.intl;
formatMessage('smth');
```

Reply:
done